CONTRIBUTING.adoc

```diff
@@ -1,13 +1,46 @@
-= Code formatting guidelines
+= Contributing to Spring Cloud GCP
+
+We graciously welcome contributions to Spring Cloud GCP.
+
+== Building the Project
+
+The first step to contributing is to fork the repository and clone it onto your machine.
+
+1. Ensure you have Java version 8 or later installed on your machine.
+
+2. Try running some tests.
++
+Run `./mvnw clean test` in the root directory of the project to run the tests.
+The `./mvnw` is a self-contained https://maven.apache.org/[Maven] wrapper that allows you to build the project without having Maven installed on your local machine.
++
+You can run the tests of a specific module by using the `-f` flag like this: `./mvnw clean test -f spring-cloud-gcp-pubsub`
+
+3. (Optional) Install the https://cloud.google.com/sdk/docs/[Google Cloud SDK].
```

Comment:
For this to be true, we need to guard the emulator-based tests behind a flag.

Reply:
I ran `./mvnw test` in a clean docker container and it worked for me. It looks like the emulator tests are already disabled by default; I get a message that looks like this: 

```
[INFO] Running org.springframework.cloud.gcp.stream.binder.pubsub.PubSubMessageChannelBinderEmulatorTests
15:47:32.591 [main] WARN  o.s.c.g.s.b.pubsub.PubSubEmulator - PubSubEmulator rule disabled. Please enable with -Dit.pubsub-emulator.
```

Reply:
Hmm, not sure why @artembilan had an issue when running the build locally then.

Reply:
I have problems only when emulator-specific tests are not guarded.
I guess everyone else without Google Cloud SDK will fail the same way...

Reply:
Lol, I think Mike meant to mention Elliotte right? And not Artem?

Reply:
Ah, you're right. @elharo